CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 #   02110-1301, USA.
 message("Entering ${CMAKE_CURRENT_SOURCE_DIR}")
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
shouldn't we reflect this change in the RPM BuildRequires and the Debian pendant, too?